azure-pipelines.yml

```diff
@@ -14,5 +14,5 @@ steps:
 - script: pip install -r requirements.txt
   displayName: 'Install required packages'
 
-- script: python -m pytest test_fairlearn_smoke.py
-  displayName: 'Run smoke tests'
+- script: python -m pytest test/test*.py
```

Comment:
This doesn't work in Powershell, but does appear to work in the build

Reply:
Probably needs ./test/test*.py ?

Reply:
Actually just running python -m pytest ./test should be sufficient

Reply:
Is PyTest smart enough to cope with files which don't contain tests? I'm thinking that we might end up with test utility files in this directory.

Reply:
It certainly should be. We've used that all over the place.

Reply:
Presumably we just need to avoid having methods start with `test_` to get that working (ISTR the non-pytest script confuses pytest for that reason)

Reply:
Ok I"m confused. I thought they HAVE to start with test_?

Reply:
I believe that PyTest requires methods to start with `test_` in order to pick them up. So if one of our utility functions starts with `test_` we could be in trouble. It's just where we put the naming convention entirely inside the files, or also have one for the filenames themselves. On reflection, I think that the former (as you've suggested) is the right one.